data/electronToChromium.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "1.5": 54,
```

Comment:
Is this the best way to track versions or is there a place for this already? cc @kevinsawicki